src/ChatWindow.test.js

```diff
@@ -139,7 +143,7 @@ it("handles the user's input as they are typing their request (before submission
   });
 
   const expectedUserRequestToAlexa = "mock request";
-  const preventDefaultSpy = jest.fn();
+  preventDefaultSpy = jest.fn();
```

Comment:
This is not needed. You can use the instance variable.

Reply:
Done.